app/code/community/Reverb/ReverbSync/Helper/Data.php

```diff
@@ -281,6 +281,7 @@ protected function _getCurlResource($url, $options_array = array())
         $options_array[CURLOPT_SSL_VERIFYPEER] = 0;
         $options_array[CURLOPT_HEADER] = 0;
         $options_array[CURLOPT_RETURNTRANSFER] = 1;
+        $options_array[CURLOPT_USERAGENT] = 'Reverb-Magento MagentoVersion=community-1.9';
```

Comment:
What would be the format for this? 

'Reverb-Magento MagentoVersion=community-1.9 domain=foo.com';

As above?